pkgs/development/python-modules/qiskit-aer/default.nix

```diff
@@ -62,7 +62,12 @@ buildPythonPackage rec {
   ];
 
   postPatch = ''
-    substituteInPlace setup.py --replace "'cmake!=3.17,!=3.17.0'," ""
+    substituteInPlace setup.py \
+      --replace "'cmake!=3.17,!=3.17.0'," "" \
+      --replace "'pybind11', min_version='2.6'" "'pybind11'" \
+      --replace "pybind11>=2.6" "pybind11" \
+      --replace "scikit-build>=0.11.0" "scikit-build" \
+      --replace "min_version='0.11.0'" ""
   '';
```

Comment:
Why got those dependencies removed?

Reply:
This is a carry-over from my NUR build, where I support nixos [20.03, 20.09]. The min versions don't apply here b/c the ``nixpkgs`` versions are higher than the min requirements. Good spot.